agents/mysql/perfschema/perfschema.go

```diff
@@ -217,6 +219,7 @@ func (m *PerfSchema) getNewBuckets(periodStart time.Time, periodLengthSecs uint3
 
 			if !m.disableQueryExamples && esh.SQLText != nil {
 				b.Common.Example = *esh.SQLText
+				b.Common.IsTruncated = strings.HasSuffix(*esh.SQLText, event.TruncatedExampleSuffix)
```

Comment:
Are we getting truncated query from perfschema?

Reply:
please read this https://books.google.com.tr/books?id=KSfXDwAAQBAJ&pg=PA103&lpg=PA103&dq=performance+schema+truncate+example+max+length&source=bl&ots=mPe5PbygAP&sig=ACfU3U0cvpm01BE4hzD_bndFsw020-lZGw&hl=en&sa=X&ved=2ahUKEwjyqrL7ufTpAhUPyaYKHY_FB3cQ6AEwAHoECAkQAQ#v=onepage&q=performance%20schema%20truncate%20example%20max%20length&f=false
It might help.